docs/overview/component/index-manager.md

```diff
@@ -0,0 +1,43 @@
+# Vald Index Manager
+
+Vald Index Manager is the component that controls the indexing process of Vald Agent Pods in the same Vald cluster.
+
+## Responsibility
+
+Vald Index Manager has a unique and simple role in preventing stop-the-world.
+
+It requires `Vald Discoverer` to fulfill its responsibility.
+
+## Features
+
+This chapter shows the main features to fulfill Vald Index Manager’s role.
+
+### Syncing data from Vald Discoverer
+
+Vald Index Manager gets the IP addresses of each Vald Agent pod from Vald Discoverer when its container starts.
+
+In addition, when IP address changes, Vald Index Manager gets the new IP address from Vald Discoverer.
+
+Vald Index Manager uses these for the controlling indexing process.
+
+### Controlling Indexing process
+
+When Vald Agent Pod creates or saves indexes on its container memory, it can not run the search process with the request query.
```

Comment:
```suggestion
When Vald Agent Pod creates or saves indexes on its container memory, it blocks all searching requests from the user and an error will be returned.
```

Reply:
```suggestion
When Vald Agent Pod creates or saves indexes on its container memory, it blocks all searching requests from the user and returns an error instead of a search result.
```